### DISCUSSION TITLE
Wipe out first week?

### DISCUSSION THREAD

**Discussion by EElks** [110yards/fantasy#183] [Q&A] [answered]
Hi Michael,

We were busy with the Oilers playoff run and didn't get to our draft until tonight-- the Sunday after the first week. Immediately following the draft's completion, the website automatically tabulated the scores for the first week--meaning that guys didn't set their lineups for injuries and byes. It wasn't a fair week. Is it possible for you to delete the first week of scoring in our league? That is, the first head to head week not to count? Our league is called, 3rd and Gamble.

**Comment by EElks** [comment]
Thanks Michael!  It's much appreciated! The first week results were wiped out.

**Comment by mdryden** [comment, marked as answer]
Haha, you're welcome except I didn't do anything.

This is a quirk of what happens when you finish the draft late.  At the end of the draft, your league start week is set to the current week, then the site checks if any games have already started, and if so, it sets your start week to next week.

Nothing stops the scores from appearing during the first week, as you saw, but the results don't actually start getting counted until your start week.

I think you're the first one to encounter this - most people see the effect of the start week simply by starting late and doing the draft before a later week starts, so I'm glad this worked out properly for you.

**Comment by EElks** [comment]
Perfect.  Either way, it worked out for us! Thanks Michael.
Mwtt
